easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2017b.eb

```diff
@@ -0,0 +1,32 @@
+name = 'netCDF'
+version = '4.5.0'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf-c/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f7d1cb2a82100b9bf9a1130a50bc5c7baf0de5b5022860ac3e09a0a32f83cf4a']
+
+dependencies = [
+    ('HDF5', '1.10.1'),
+    ('cURL', '7.56.1'),
+]
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+    ('Doxygen', '1.8.13'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF ",
+    "-DBUILD_SHARED_LIBS=ON ",
```

Comment:
I assume this is right, because it was in the previous versions. Just looks weird...

Reply:
The intention here is to build both static and shared libraries. I think they make you pick, and so if you want both, you need to do two `cmake - make - make install` cycles, which is exactly what this does: we configure once with `-DBUILD_SHARED_LIBS=OFF` (and then `make` + `make install`), and then do another cycle when configuring with `-DBUILD_SHARED_LIBS=ON`...